fence/auth.py

```diff
@@ -173,7 +249,10 @@ def login_required(scope=None):
     def decorator(f):
         @wraps(f)
         def wrapper(*args, **kwargs):
+            logger.info("Decorator login_required wrapper")
```

Comment:
either change all these to logger.debug so that they are disabled in prod running or cancel them to keep the log cleaner.